stylesheets/commons/Teamcard.css

```diff
@@ -139,20 +139,19 @@ Author(s): ???
 }
 
 .teamcard-toggle-button {
+	display: flow-root;
```

Comment:
iirc not supported in ie 11
if we still want to support ie11 ...
i personally think anyone still using ie11 is at fault theirselfs